src/app/(authenticated)/comic/[slug]/episodes/page.tsx

```diff
@@ -0,0 +1,41 @@
+import { fetchComic } from '@/app/lib/api/comic/queries'
+import { fetchComicIssues } from '@/app/lib/api/comicIssue/queries'
+import { getAccessToken } from '@/app/lib/utils/auth'
+import { ComicBanner } from '@/components/comic/Banner'
+import { ComicHeader } from '@/components/comic/Header'
+import { DefaultComicIssueCard } from '@/components/comicIssue/cards/DefaultCard'
+import { BaseLayout } from '@/components/layout/BaseLayout'
+import { Tabs } from '@/components/shared/Tabs'
+import { getComicPageTabs } from '@/constants/tabs'
+import React from 'react'
+
+type Props = {
+  params: {
+    slug: string
+  }
+}
+
+export default async function ComicEpisodesPage({ params: { slug } }: Props) {
+  const comic = await fetchComic({ accessToken: getAccessToken(), slug })
+  const tabs = getComicPageTabs(slug)
+  const comicIssues = await fetchComicIssues({ comicSlug: slug, skip: 0, take: 6 })
+
+  if (!comic || !comic.stats) {
```

Comment:
move this condition to be below fetchComic request so we don't do unnecessary api call to fetch comic issue

```
  const comic = await fetchComic({ accessToken: getAccessToken(), slug })
  if (!comic || !comic.stats) {
    return null
  }

  const tabs = getComicPageTabs(slug)
  const comicIssues = await fetchComicIssues({ comicSlug: slug, skip: 0, take: 6 })
 ```